core/app/models/spree/property.rb

```diff
@@ -19,9 +19,11 @@ class Property < Spree::Base
 
     self.whitelisted_ransackable_attributes = ['presentation']
 
-    def uniq_values
+    def uniq_values(product_properties_scope: nil)
       Rails.cache.fetch("property-uniq-values/#{cache_key_with_version}") do
```

Comment:
@mad-eel we need to include that scope in the cache key or we can remove that cache statement for now

Reply:
@damianlegawiec Thanks, good catch! I corrected caching a bit, but it'll only work for calls without scope. Caching scoped calls seem much more tricky, so maybe let's leave it for now.